hexa/ui/datagrid.py

```diff
@@ -309,29 +307,11 @@ def context(self, model: DjangoModel, grid: Datagrid):
         return data
 
 
-class BooleanColumn(Column):
-    """Simple boolean column"""
-
-    def __init__(self, *, value=None, **kwargs):
-        super().__init__(**kwargs)
-
-        self.value = value
-
-    @property
-    def template(self):
-        return "ui/datagrid/column_boolean.html"
-
-    def context(self, model: DjangoModel, grid: Datagrid):
-        return {
-            "value": self.get_value(model, self.value, container=grid),
-        }
-
-
 class DateColumn(Column):
     """Date column, with one or two rows"""
 
     def __init__(
-        self, *, date=None, date_format="timesince", secondary_text=None, **kwargs
+        self, *, date=None, date_format="%Y-%m-%d %H:%M", secondary_text=None, **kwargs
```

Comment:
Don't change the default value of `date_format`. Other grids use this column as well and rely on the existing default.

In this case, your goal it to use a specific format, not change the default for all other use cases.